pkg/sql/schemachanger/scplan/internal/scstage/build.go

```diff
@@ -361,7 +358,7 @@ func (sb stageBuilder) nextTargetState(t currentTargetState) currentTargetState
 		} else {
 			next.hasOpEdgeWithOps = true
 		}
-	}
+	}*/
```

Comment:
Just get rid of this, as suggested in a previous comment, along with `hasOpEdgeWithOps`.